### PR TITLE
Added the ctor to create a sfColor from an sfUint32

### PR DESCRIPTION
**Pull request by zsbzsb**
https://github.com/LaurentGomila/SFML/pull/722

### CODE REVIEW

File: include/SFML/Graphics/Color.h [hunk 1]
```diff
@@ -81,6 +81,24 @@ CSFML_GRAPHICS_API sfColor sfColor_fromRGB(sfUint8 red, sfUint8 green, sfUint8 b
...
 
 ////////////////////////////////////////////////////////////
+/// \brief Construct the color from 32-bit unsigned integer
+///
+/// \param color Number containing the RGBA components (in that order)
+///
+/// \return sfColor constructed from the 32-bit unsigned integer
+///
+////////////////////////////////////////////////////////////
+CSFML_GRAPHICS_API sfColor sfColor_fromUInt(sfUint32 color);
```
Comment (LaurentGomila) [inline include/SFML/Graphics/Color.h]: Why not ```sfColor_fromInteger```?

Comment (zsbzsb) [inline include/SFML/Graphics/Color.h]: No real reason why.... other than keeping you on your toes :wink:

**Comment by zsbzsb** [comment]
Branch has been updated.

**Comment by zsbzsb** [comment]
Just re-reading the commit message, what kind of English was I thinking when I wrote that?

>Added the ctor to create a sfColor from an sfUint32

Should be

>Added a ctor to create a sfColor from a sfUint32

**Comment by LaurentGomila** [comment]
:grin:

**Pull request merged**